### DISCUSSION TITLE
Multimarkdown Graphics

### DISCUSSION THREAD

**Discussion by iwelch** [jgm/pandoc#9702] [Q&A]
The `-r multimarkdown` image format is quite flexible insofar as options are concerned, including unused ones.  Is there a document that tells me which of the mmd options are picked up by pandoc (and how they translate / are passed either to figure, a sub of it (like caption), or includegraphics/img).

PS: What in multimarkdown escapes raw tex commands, and how do I reenable it?

**Comment by jgm** [comment]
Try adding `+raw_tex`: `-f markdown_mmd+raw_tex`
